notebooks/multiple_lens_injection.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>perhaps describe what this line should display?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/sim-pipeline/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>